observe-api/go/go.mod

```diff
@@ -0,0 +1,3 @@
+module dylibso.com/observe_api
```

Comment:
I think this should be `github.com/dylibso/observe-sdk/observe-api/go`, we _can_ add our own vanity import path, but we need to set up some redirects... probably not worth the trouble yet though.